collector/droplet.go

```diff
@@ -85,16 +85,45 @@ func (c *DropletCollector) Describe(ch chan<- *prometheus.Desc) {
 func (c *DropletCollector) Collect(ch chan<- prometheus.Metric) {
 	ctx, cancel := context.WithTimeout(context.Background(), c.timeout)
 	defer cancel()
-	droplets, _, err := c.client.Droplets.List(ctx, nil)
-	if err != nil {
-		c.errors.WithLabelValues("droplet").Add(1)
-		level.Warn(c.logger).Log(
-			"msg", "can't list droplets",
-			"err", err,
-		)
+
+	// create a list to hold our droplets
+	list := []godo.Droplet{}
+
+	// create options. initially, these will be blank
+	opt := &godo.ListOptions{}
+
+	for {
+		droplets, resp, err := c.client.Droplets.List(ctx, opt)
+
+		if err != nil {
```

Comment:
could you make sure to increment the metric too?
```suggestion
		if err != nil {
			c.errors.WithLabelValues("droplet").Add(1)
```

Same for the error further down :)